_components/headers.md

```diff
@@ -5,8 +5,107 @@ title: Headers & Navigation
 ---
 
 <div class="preview">
-  <!-- Add HTML markup for example here -->
-  <img src="{{ site.baseurl }}/assets/img/static/HeaderNav_FullUI_v1-930width.png">
+  
+  <h2 class="usa-heading">Side Nav</h2>
+
+  <div class="usa-grid">
+    <aside class="usa-width-one-fourth">
+      <ul class="usa-sidenav-list">
```

Comment:
Bc it's applied to the list class and we have sublists nested beneath it (children and grandchild). So thought we should mark this out.